.travis.yml

```diff
@@ -11,26 +9,43 @@ matrix:
     - php: 7.1
       env: PHPSTAN=true
 
-cache:
-  directories:
-    - $HOME/.composer/cache
+env:
+  global:
+    - VERSION=$(echo $TRAVIS_TAG | cut -c 2-10)
 
 install:
-  - composer install --no-interaction
-  - if [[ $ECS != "" ]]; then composer create-project symplify/easy-coding-standard temp/ecs; fi
+- composer install --no-interaction --prefer-dist -o
+- if [[ $ECS != "" ]]; then composer create-project symplify/easy-coding-standard temp/ecs; fi
+
+jobs:
+  include:
+    - stage: Test
+      script:
+        - vendor/bin/phpunit
+        - if [[ $ECS != "" ]]; then temp/ecs/bin/ecs check src tests; fi
```

Comment:
Think you have to add a new include for these subtasks like

```
- stage: lint
   script: vendor/bin/phpstan analyse src --level max --configuration phpstan.neon;

- stage lint
  script: temp/ecs/bin/ecs check src tests;
  before_script: composer create-project symplify/easy-coding-standard temp/ecs;
```
The if statemens are just a hack to simulate stages. which are now available in travis